src/main/scala/org/broadinstitute/dsde/rawls/util/WithSupport.scala

```diff
@@ -52,3 +52,47 @@ trait MethodWiths {
     }
   }
 }
+
+trait UserWiths {
+  val dataSource: SlickDataSource
+
+  import dataSource.dataAccess.driver.api._
+
+  def withUserInTransaction[T](rawlsUserRef: RawlsUserRef)(op: RawlsUser => Future[T])(implicit executionContext: ExecutionContext): Future[T] = {
```

Comment:
Alternatively, just call this one `withUser` and make the other one `withUserUsingTransaction`.